### PR TITLE
Add support for org tag to create and update operations

### PR DESCRIPTION
**Pull request by fishnix**
Org is required for cost information and other things.  It should be applied to all resources by all of our APIs.

### CODE REVIEW

File: actions/databases.go [hunk 1]
```diff
@@ -486,3 +486,22 @@ func DatabasesDelete(c buffalo.Context) error {
 
 	return c.Render(200, r.JSON(output))
 }
+
+// normalizTags strips the org from the given tags and ensures it is set to the API org
```
Comment (tenyo) [inline actions/databases.go]: 👀

File: config/config.example.json [hunk 2]
```diff
@@ -21,5 +21,6 @@
       }
     }
   },
-  "token": "TOKEN"
+  "token": "TOKEN",
+  "org": "localdev"
```
Comment (tenyo) [inline config/config.example.json:25]: I don't think we have a deco config file here, but we'll need to add this to the Jenkins jobs

**Review by tenyo — COMMENTED** [review]
Looks good 👍 
Do we need to add the org tag to existing databases?

File: config/config.example.json (hunk 2 above)
Comment (fishnix) [reply config/config.example.json:25]: yeah I looked for it and then remember we just use jenkins for this one

**Comment by damnski** [comment]
:+1:

**Comment by fishnix** [comment]
@tenyo yes, we should go through and tag the existing db's with the org 👍

**Pull request merged**